osbuild/util/pe32p.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/python3
+"""
+Utility functions to inspect PE32+ files
+
+
+"""
+
+import enum
+import io
+import os
+import struct
+import sys
+from collections import namedtuple
+from typing import Iterator, List, Optional, Union
+
+PathLike = Union[str, bytes, os.PathLike]
+
+CoffFormat = "4sHHIIIHH"
+CoffHeader = namedtuple(
+    "CoffHeader",
+    "Signature "
+    "Machine "
+    "NumberOfSections "
+    "TimeDateStamp "
+    "PointerToSymbolTable "
+    "NumberOfSymbols "
+    "SizeOfOptionalHeader "
+    "Characteristics"
```

Comment:
Do you prefer the white-space separated fields over an array of field-names?

Reply:
Doh, no. I looked at the wrong docs / example. Let me fix that. Thanks.